states/menu.py

```diff
@@ -5,22 +5,29 @@
 
 
 class MenuOptions(StateOptions):
+    """Enum values for options available in the menu"""
+
     BATTLE = 1
     TEAM = 2
     EXIT = 3
 
 
 class Menu(State):
+    """Displays, holds, and reads options for Menu state change"""
```

Comment:
Minor: make this more consistent with `state/battle.py`'s docstring
`"""Game state for player battling"""`
so maybe go with
`"""Game state for main menu"""`
The `Displays, holds, and reads options for state change` is a good description, so maybe move that to the `State` class in `states/state.py` since essentially all game states will be exhibiting this behavior.